src/Topics/Lebensohl.hs

```diff
@@ -371,12 +371,83 @@ bid3NWithoutStopper = let
                       <~ T.allVulnerabilities
 
 
+cueBidWithStopper :: Situations
+cueBidWithStopper = let
+    sit (overcall, relay, bid) = let
+        action = do
+            setOpener T.North
+            Leb.b1N
+            _ <- overcall
+            withholdBid bid
+        explanation =
+            "Partner opened a strong " .+ Leb.b1N .+ ", and RHO " .+
+            "interfered with the auction. We'd prefer to play in a " .+
+            "major-suit fit if we have one, but can tolerate " .+
+            T.Bid 3 T.Notrump .+ " without one because we have a stopper " .+
+            "in the opponent's suit. Relay through " .+
+            relay .+ ", then bid " .+ bid .+ " as Stayman. Partner can " .+
+            "bid the correct game, based on whether we've got a major-suit fit."
```

Comment:
Mention how this bid is forcing, so we have the ability to investigate slam if we've got extra strength. Same comment below.